src/test/scala/chiselTests/BlackBox.scala

```diff
@@ -2,11 +2,11 @@
 
 package chiselTests
 
+import circt.stage.ChiselStage
 import chisel3._
 import chisel3.experimental._
 import chisel3.testers.{BasicTester, TesterDriver}
 import chisel3.util._
-import circt.stage.ChiselStage
```

Comment:
curious why you moved the imports around here?

Reply:
There are comments buried above. `chisel3.util.circt` (added by this PR) confuses `import circt.stage.ChiselStage`. Andrew fixed this with `import _root_.circt.stage.ChiselStage`. Jack suggested moving the imports instead.

Reply:
Because apparently scala doesn't search from the root first for new imports, and adding `_root_.` was kind of ugly.